### PR TITLE
FIX: Crashes for models that have change_form_template set

### PR DESCRIPTION
**Pull request by danialmalik**
## Description
`change_form_template` is not supported by `MassAdmin` right now. If any model admin has this value set, the app crashes on the mass edit page.

Since the change form and mass edit for serve two different purposes, it makes sense to separate the variable that can be used to override their templates.

### Testing
Testing instructions can be found at https://github.com/unhaggle/tier3/pull/5681

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,21 +18,4 @@ after_success:
...
-      env: DJANGO="Django>=2.0,<2.1"
-    - python: "2.7"
-      env: DJANGO="Django>=2.1,<2.2"
-    - python: "2.7"
-      env: DJANGO="Django>=2.2,<3.0"
-    - python: "2.7"
-      env: DJANGO="Django>=3.0,<3.1"
-    - python: "2.7"
-      env: DJANGO='https://github.com/django/django/archive/master.tar.gz'
+     - env: DJANGO="Django>=3.0,<3.1"
```
Comment (danialmalik) [inline .travis.yml:21]: We're using django 2.2.

**Review by ibaguio — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
   - "3.7"
 env:
-  - DJANGO="Django>=1.8,<1.9"
-  - DJANGO="Django>=1.9,<1.10"
-  - DJANGO="Django>=1.10,<1.11"
-  - DJANGO="Django>=1.11,<2.0"
-  - DJANGO="Django>=2.0,<2.1"
   - DJANGO="Django>=2.1,<2.2"
   - DJANGO="Django>=2.2,<3.0"
   - DJANGO="Django>=3.0,<3.1"
```
Comment (dhruvsingh) [inline .travis.yml:8]: Do we want to pin the version for Django to 2.2 here since we're using that version?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
-[![Build Status](https://travis-ci.org/burke-software/django-mass-edit.svg?branch=master)](https://travis-ci.org/burke-software/django-mass-edit)
-[![Coverage Status](https://coveralls.io/repos/burke-software/django-mass-edit/badge.svg?branch=master&service=github)](https://coveralls.io/github/burke-software/django-mass-edit?branch=master)
+# Django Mass Edit
+
+[![Build Status](https://app.travis-ci.com/unhaggle/django-mass-edit.svg?branch=master)](https://travis-ci.org/burke-software/django-mass-edit)
```
Comment (dhruvsingh) [inline README.md:3]: Shall we remove the original authors build status from here and use a separate comment line for "original authors build status"?

File: setup.cfg [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.2.0
+current_version = 3.3.1uh
```
Comment (dhruvsingh) [inline setup.cfg:2]: Why is this `3.3.1`?
Should it be `3.3.0`?

File: setup.py [hunk 5]
```diff
@@ -2,21 +2,20 @@
 
 setup(
     name = "django-mass-edit",
-    version="3.3.0",
+    version="3.3.1uh",
```
Comment (dhruvsingh) [inline setup.py:5]: Is the version in `setup.cfg` incorrect?
I now understand why the version number is `3.3.1` but above its bumped from `3.2.0` -> `3.3.1`

**Review by dhruvsingh — APPROVED** [review]
Please see questions.
LGTM otherwise

File: setup.py (hunk 5 above)
Comment (danialmalik) [reply setup.py:5]: Yep, looks like it. the versions were different in setup.cfg and setup.py.

but looking at your comments, I think `3.3.0` would be a good choice actually.

File: README.md (hunk 3 above)
Comment (danialmalik) [reply README.md:3]: That's a good idea actually.

**Pull request merged**